packages/flutter/lib/src/foundation/bitfield_unsupported.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Unsupported.
+int get kMaxUnsignedSMI => throw UnsupportedError('Not supported in environments without 64 bit ints');
+
+/// A BitField over an enum (or other class whose values implement "index").
```

Comment:
Should the doc comments here and/or in the other implementation mention the conditions for when this thing is actually supported?

(I am also wondering how dartdocs deals with conditional imports? Does it only show the dart docs for one of the impls? If so, which one? Does this thing show up twice in dart docs now?)

Reply:
I don't think they should show up at all, but I'll need to test it.

Either the conditionally imported file has the same API, or we're in a world of hurt. That would mean all documentation should be on the main (non-conditional) implementation. I'm not sure if that means we should leave stub comments or something on the conditional file.